lms/static/sass/shared/_header.scss

```diff
@@ -3,7 +3,7 @@
 
 header.global {
   @extend %ui-depth1;
-  border-bottom: 1px solid $m-gray;
+  border-bottom: 4px solid $courseware-border-bottom-color;
```

Comment:
the use of the courseware-border variable is intended to use the existing theming work. It does make the openedx header look closer to the edx, but I'm happy to revert the change.